packages/client/hmi-client/src/components/data-explorer/search-bar.vue

```diff
@@ -122,6 +126,16 @@ watch(suggestedItems, (newItems) => {
 	color: var(--text-color-secondary);
 }
 
+.pi-image {
```

Comment:
Can you use a custom class name for this behaviour, since `pi-image` should just be used to define what icon is displayed